### ISSUE TITLE
How to get value of the edge storage key without subquery

### ISSUE THREAD

**Issue by Siceberg** [facebook/ent#1170]
For example, can I get the `owner_id` while getting the `Pet` object without subquery. I know "Eager Loading" can be used to achieve the goal, but the subquery is not necessary because the fields I need are already in the car table, and this is not efficient.

**Comment by a8m** [comment]
Hey @Siceberg and thanks for opening this issue.

This is duplicate of #869 and should be addressed soon with https://github.com/facebook/ent/pull/1046.

Closing, but I'll will update this issue once https://github.com/facebook/ent/pull/1046 is merged 😃

**Comment by Siceberg** [comment]
Hi @a8m , this is great, thanks for your reply
